Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -128,9 +135,9 @@ public final class SwiftPMWorkspace {
     // FIXME: make these configurable
```

Comment:
should triple be configurable here? if not I guess we can remove the FIXME

Reply:
Removing the FIXME sounds fine!  We may someday want to customize the triple, but not for now.